inference-engine/thirdparty/clDNN/src/kernel_selector_helper.cpp

```diff
@@ -271,6 +271,9 @@ kernel_selector::weights_layout to_weights_layout(format f) {
         case format::os_is_y_x8_osv8_isv4_swizzled_by_4:
             return kernel_selector::weights_layout::os_is_y_x8_osv8_isv4_swizzled_by_4;
         case format::bfzyx:
+            return is_grouped ? kernel_selector::weights_layout::goiyx : kernel_selector::weights_layout::oizyx;
+        case format::bfwzyx:
+            return kernel_selector::weights_layout::goizyx;
```

Comment:
Maybe it's better to add some exception in case when `format::bfwzyx` and `is_grouped == false` to avoid ambiguity in the feature?